src/libtiled/mapreader.h

```diff
@@ -95,6 +97,9 @@ class TILEDSHARED_EXPORT MapReader
      */
     QString errorString() const;
 
+	int GetNumTilesetSourceFiles();
+	QString* GetTilesetSourceFiles();
```

Comment:
Since these functions are not modifying the object, they should be marked as `const`.